certbot/tests/reporter_test.py

```diff
@@ -39,8 +42,10 @@ def test_no_tty_print_empty(self):
             self.reporter.print_messages()
         self.assertEqual(sys.stdout.getvalue(), "")
 
-    def test_atexit_print_messages(self):
+    @mock.patch('certbot.reporter.os.getpid')
+    def test_atexit_print_messages(self, mock_getpid):
         self._add_messages()
+        mock_getpid.return_value = self.rep_module.INITIAL_PID
         self.reporter.atexit_print_messages()
```

Comment:
Instead of setting `self.rep_module` to the `reporter` module in `setUp` (which is specifically imported that way to minimize the amount of the tests that have access to the module) you can also just mock out `INITIAL_PID` so you can control both values. You can do this with a context manager like this:
```
with mock.patch('certbot.reporter.INITIAL_PID', 5):
```
or as a decorator like this:
```
    @mock.patch('certbot.reporter.INITIAL_PID', 5)
    @mock.patch('certbot.reporter.os.getpid')
    def test_atexit_print_messages(self, mock_getpid):
```
Notice in the latter example, an additional argument isn't added to the function.